### DISCUSSION TITLE
Should commercial only project unioffice be removed since there is requirement for OSI approved license?

### DISCUSSION THREAD

**Discussion by isopov** [avelino/awesome-go#5191] [Q&A]
- unioffice is listed as a single option for .docx and .pptx 
- currently it is commercially licensed only - https://github.com/unidoc/unioffice/blob/master/LICENSE.md
- it seems that when it was added it had AGPL license also which is listed as acceptable, but currently it has no such option

**Comment by yue1003** [comment]
NB

**Comment by phanirithvij** [comment]
Good question, @avelino need your input here.

I am in favor of removal.
Edit: see discussion below.

**Reply by avelino** [reply]
> this relicensing point is complicated and we probably can't automatically keep track of all the projects :(

about the unioffice project we're talking about in this thread:
since it doesn't have an OSI license and is a requirement to be in awesome-go, we should remove it

do you send PR removing it?

@phanirithvij can you update our contribution guide to say that projects relicensed for non-OSI licenses will be removed?

**Reply by phanirithvij** [reply]
That's a slippery slope, should we start removing BSL/BUSL, SSPL, FSL licensed projects as well? Where should we draw the line?

Should we maybe let it be and update its category to be in "Source Available", or edit the description to mention the licence in brackets? Eg. unioffice - ...... (licensed Proprietary and source available)
Need to do it for all non-osi compliant projects.

**Reply by avelino** [reply]
I would put these projects in a category of proprietary licenses, do you think that makes sense?

> remember that I'm not the owner of the truth, awesome-go is a project of ours and we're maintaining it together, I need help to define the best way to solve this problem
